service-registry/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM openjdk:17-jdk-slim
+COPY build/libs/*.jar app.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)